bridgesample/src/main/java/com/livefront/bridgesample/scenario/activity/NonBridgeLargeDataActivity.kt

```diff
@@ -0,0 +1,66 @@
+package com.livefront.bridgesample.scenario.activity
+
+import android.content.Intent
+import android.graphics.Bitmap
+import android.os.Bundle
+import android.view.MenuItem
+import com.evernote.android.state.State
+import com.livefront.bridgesample.R
+import com.livefront.bridgesample.base.NonBridgeBaseActivity
+import com.livefront.bridgesample.util.generateNoisyStripedBitmap
+import kotlinx.android.synthetic.main.activity_large_data.generateDataButton
+import kotlinx.android.synthetic.main.activity_large_data.headerText
+import kotlinx.android.synthetic.main.activity_large_data.imageView
+import kotlinx.android.synthetic.main.activity_large_data.navigateButton
+import kotlinx.android.synthetic.main.activity_large_data.statusText
+import kotlinx.android.synthetic.main.basic_toolbar.toolbar
+
+class NonBridgeLargeDataActivity : NonBridgeBaseActivity() {
+    @State
+    lateinit var bitmap: Bitmap
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_large_data)
+        setSupportActionBar(toolbar)
+        supportActionBar!!.apply {
+            setDisplayHomeAsUpEnabled(true)
+            setHomeButtonEnabled(true)
+            setTitle(R.string.non_bridge_large_data_screen_title)
+        }
+
+        headerText.setText(R.string.non_bridge_large_data_header)
+
+        if (this::bitmap.isInitialized) {
+            // The Bitmap is restored from a saved state. We can just show it directly
```

Comment:
Minor -- add a period to the end of the second sentence.

Reply:
Sure.